Dockerfile

```diff
@@ -11,7 +11,8 @@ pip3 install \
         certifi==2020.12.5 chardet==4.0.0 idna==2.10 urllib3==1.26.4 \
         six==1.15.0 msgpack==1.0.2 \
     influxdb_client==1.24.0 rx==3.2.0 \
-    yagrc==1.1.1 grpcio-reflection==1.36.1 protobuf==3.15.6
+    yagrc==1.1.1 grpcio-reflection==1.36.1 protobuf==3.15.6 \
+    typing-extensions>=4.3.0
```

Comment:
For `Dockerfile`, the packages should be pinned to a specific version (so `==` instead of `>=`), to ensure the image generated is a known good set of package versions.

`requirements.txt` and `setup.cfg` are left more flexible to play more nicely with other packages a user has installed, at the risk of breaking if a bad or interface-breaking package is pushed.